pkg/entity/id.go

```diff
@@ -9,6 +9,28 @@ import (
 	"strings"
 )
 
+// Entity information.
+type Entity struct {
+	Key
+	ID
+}
+
+// New will create a new Entity object.
+func New(entityKey Key, entityID ID) Entity {
+	return Entity{
+		Key: entityKey,
+		ID:  entityID,
+	}
+}
+
+// NewWithoutID will create an empty Entity object without entityID.
+func NewWithoutID(entityKey string) Entity {
```

Comment:
Why are we use a `string` hare and a `Key` in a `New()`? Is it on purpose or typo?